AndroidTrainingModule/res/layout/relative_layout_activity.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent" >
+    
+    <TextView 
+        android:id="@+id/main_textView"
+        android:gravity="center"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:textSize="30sp"
+        android:text="@string/hello_world"/>
+
+    <EditText
+        android:id="@+id/main_editText"
+        android:layout_width="match_parent"
+        android:layout_height="50dp"
+        android:layout_alignParentLeft="true"
+        android:layout_below="@+id/main_textView"
```

Comment:
Please remove the `+` symbol. This may cause issue since the id main_textView is already set before main_editText view. You may strictly use `@+id/main_textView` in setting the id or if the view to call is placed below `@+id/main_textView`